inception-project/inception
# #1047 - Add docker support

**What's in the PR**
* Docker file construction

**How to test manually**
* Run `mvn -Pdocker clean install docker:build` and then start a container of this type `docker run -v /path/on/host/inception/repository:/export -p port-on-host:8080 inceptionproject/inception-snapshots:latest`
* Run `mvn -Pdocker clean install docker:build -Ddocker.image.name="inceptionproject/inception"` and then start a container of this type `docker run -v /path/on/host/inception/repository:/export -p port-on-host:8080 inceptionproject/inception:latest`
* In both cases, check that INCEpTION started by going to `localhost:8080`

**Automatic testing**

**Documentation**
